packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -5725,6 +5725,81 @@ Expected input body:
 }
 ```
 
+### Type_Model_Templated_float32TemplatedType
+
+- Endpoint: `put /type/model/templated/float32ValuesType`
+
+Expected input body:
+
+````json
+{
+  "kind": "Float32Values",
+  "values":
+  [
+    12.34
+  ],
+  "value": 12.34
+}
+Expected response body:
+{
+  "kind": "Float32Values",
+  "values":
+  [
+    12.34
+  ],
+  "value": 12.34
+}
+
+### Type_Model_Templated_int32TemplatedType
+
+- Endpoint: `put /type/model/templated/int32ValuesType`
+
+Expected input body:
+```json
+{
+  "kind": "Int32Values",
+  "values":
+  [
+    1234
+  ],
+  "value": 1234
+}
+````
+
+Expected response body:
+
+```json
+{
+  "kind": "Int32Values",
+  "values": [1234],
+  "value": 1234
+}
+```
+
+### Type_Model_Templated_numericTemplatedType
```

Comment:
@melina5656 last nit, but otherwise looks great. Can you just rename them to `Type_Model_Templated_numeric` etc? We don't need `TemplatedType` again

Reply:
Hi @iscai-msft, If I simplify the name of the operation to `numeric`, `float32`, `int32`, it will lead to the problem of `the constraint of type parameter' T 'referencing' T 'itself`. Therefore, I would like to ask if it is possible to change the name of the operation to `numericType`, `float32Type`, `int32Type`.

Reply:
ah that's a good point, I didn't think about that. Yeah go ahead with `Templated_numericType`

Reply:
Hi @iscai-msft , I have uploaded the latest changes. If you feel that the code meets the requirements, please leave a comment in this conversation and I will merge this PR after receiving the message.